From 4c5f6d15ad2bf4a9b5dc1b9f999fb32ddf8f948a Mon Sep 17 00:00:00 2001
From: Tyler Baker <tyler@foundries.io>
Date: Mon, 24 Sep 2018 13:56:33 -0700
Subject: [PATCH] configs: dragonboard: disable autboot console check

When the UART adapter isn't active, the handoff between
LK and u-boot is stopping the autoboot process. Until the
issue is understood, lets disable boot interrupt via
the console.

Signed-off-by: Tyler Baker <tyler@foundries.io>
---
 configs/dragonboard820c_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/dragonboard820c_defconfig b/configs/dragonboard820c_defconfig
index 6b963061fd..68501075d3 100644
--- a/configs/dragonboard820c_defconfig
+++ b/configs/dragonboard820c_defconfig
@@ -8,6 +8,7 @@ CONFIG_DISTRO_DEFAULTS=y
 CONFIG_NR_DRAM_BANKS=2
 CONFIG_USE_BOOTARGS=y
 CONFIG_BOOTARGS="console=ttyMSM0,115200n8"
+CONFIG_BOOTDELAY=-2
 # CONFIG_USE_BOOTCOMMAND is not set
 CONFIG_MISC_INIT_R=y
 # CONFIG_DISPLAY_CPUINFO is not set
-- 
2.17.1

